contents/indexing.md

```diff
@@ -94,8 +94,8 @@ sco("grades_2020()"; process=without_caption_label)
 Add ref to multiple dispatch in the intro
 ```
 
-We can filter rows by using `filter(f::Function, df)`.
-This function is very similar to the function `filter(f::Function, V::Vector)` from Julia itself.
+We can filter rows by using `filter(source => f::Function, df)`.
+Note how this function is very similar to the function `filter(f::Function, V::Vector)` from Julia itself.
```

Comment:
why is this important? what's the different, is it worth saying?

Reply:
Because it explains why `filter(..., df)` and `select(df, ...)`, `subset(df, ...)` and `transform(df, ...)`.